Responsive Landing Page/dist/index.html

```diff
@@ -74,5 +74,44 @@ <h2 class="text-xl font-bold text-blue-800">"Freshly Ground, Perfectly Milled"</
      </div>
     <!-- Achievements section ends here -->
 
+    <!-- Achievements section starts here -->
+    <div class=" text-center flex flex-col items-center">
+        <div class="mt-6">
+            <h2 class="text-xl md:text-2xl lg:text-3xl font-bold lg:font-semibold text-blue-800">"Our Services"</h2>
+            <div class="w-1/2 mx-auto border-2 border-orange-400 mt-3"></div>
+        </div>
+        <div class="mt-12 sm:w-[90%] lg:w-full sm:flex sm:justify-evenly sm:gap-10 lg:gap-0 flex-wrap">
+            <div class="w-3/5 mx-auto min-w-44 sm:w-44 flex flex-col items-center mb-10 border-2 border-gray-400 rounded-md p-2 box-border">
```

Comment:
Please make border of every service box `white` on `hover`.
If possible animate is to change a little slow instead of instant change, to look nicer.

Reply:
Added hover effect on service boxes and made it transition smoothly.
Check commit 57e881e